source/class/qx/ui/mobile/dialog/Popup.js

```diff
@@ -223,9 +223,9 @@ qx.Class.define("qx.ui.mobile.dialog.Popup", {
 
           this.placeTo(computedPopupPosition.left, computedPopupPosition.top);
         }
-      } else if (this.__childrenContainer) {
-        // No Anchor
-        this._positionToCenter();
+      } else if (this.__childrenContainer && !this.__placed) {
+          // No Anchor and not placed to point manually
+          this._positionToCenter();
```

Comment:
what if the dialog has been placed but not in the center (yet)?

Reply:
if to call `show` before "placeTo" popup will have margin values and if after it wont assign margin value.

Reply:
@oetiker if the dialog has been placed what's point to place it again? I think in this case there is no needed actions to do. If the dialog was not placed at all it will be centered, if it has an anchor it will be placed to coords of the anchor widget.